README.md

```diff
@@ -1,31 +1,121 @@
+
+## Quick start
+
+1. [Install DVC](https://dvc.org/doc/install) on your system.
+2. Install [this extension] in VS Code.
+3. Follow the **Get Started** page that pops up!
+
+<!-- prettier-ignore -->
+> 💡 Feel free to try our [example DVC project] first! Opening it [with Github
+> Codespaces] will include this extension automatically.
+
+![Initial Get Started page](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/docs/walkthroughs.png)
+
+[this extension]:
+  https://marketplace.visualstudio.com/items?itemName=Iterative.dvc
+[example dvc project]: https://github.com/iterative/example-get-started
+[with github codespaces]:
+  https://docs.github.com/en/codespaces/getting-started/quickstart#creating-your-codespace
+
+### Learn more about DVC (Data Version Control)
+
+See the DVC documentation to Get Started with [Experiment Versioning] or [Data
+Management]. For deeper learning, try our [free course]!
+
+[experiment versioning]: https://dvc.org/doc/start/experiments
+[data management]: https://dvc.org/doc/start/data-management
+[free course]: https://learn.iterative.ai/
+
+<!-- [learn more]: extension/resources/walkthrough/dvc-learn-more.md -->
+
+## What you get
+
+This extension augments VS Code in the following ways:
+
+- Adds a brand-new new **[DVC View]** (<img
+  src="https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/docs/dvc.png"
+  alt="DVC logo" height="16"/> icon in the Activity Bar) with panels to
+  visualize and manage [DVC Experiments].
+- Provides special _Editors_ to manage **[Experiments]** and display **[Plots]**
+  in IDE _Tabs_.
+- Adds a **[DVC Tracked]** panel to the _Explorer_ view. This shows a tree of
+  the data files and directories tracked by DVC, including their state, and
+  options to [synchronize] them (from/to remote storage).
+- Adds a **[DVC panel]** to the _Source Control_ view to display the [workspace
+  status]. You can [restore] or reset project versions (based on the current Git
+  `HEAD` commit) as well as manage new and existing data from here.
+- Registers several **Commands** in the _[Command Palette]_ (see next section).
+- Includes a DVC channel for the _Output_ panel (useful for
+  [debugging](#debugging)).
+
+[dvc view]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/view-container.md
+[experiments]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/experiments-table.md
+[plots]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
+[command palette]:
+  https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette
+[dvc tracked]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/dvc-tracked-files.md
+[synchronize]: https://dvc.org/doc/start/data-management#storing-and-sharing
+[dvc panel]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/dvc-tracked-files.md
```

Comment:
Change resources/walkthrough links for proper docs? Rel. https://github.com/iterative/dvc.org/pull/3663